spec.bs

```diff
@@ -1592,8 +1596,8 @@ To <dfn>fetch WebAssembly</dfn> given a [=URL=] |url|:
 
 The <dfn http-header><code>Data-Version</code></dfn> HTTP response header is a
 [=structured header=] whose value must be an [=structured header/integer=].
-The <dfn http-header><code>X-protected-audience-bidding-signals-format-version</code></dfn>
-HTTP response header is a [=structured header=] whose value must be an [=structured header/integer=].
+The <dfn http-header><code>X-fledge-bidding-signals-format-version</code></dfn> HTTP response header

```

Comment:
I don't think we should be re-introducing the old header names here and on line 1639.  I understand we don't need web developers to change to the new names (as they'll hopefully be going away shortly), but I think we even less so want them to change from the new names (if they read the spec in the past 5+ months) to the old names as we'll definitely support the new name as long as we support the old name (hopefully not long).

Reply:
I don't think we ever supported the "X-protected-audience-bidding-signals-format-version" header which is currently in the spec. The old supported name (which people are using) is "X-fledge-bidding-signals-format-version", and the new name is "Ad-Auction-Bidding-Signals-Format-Version".

I think someone changed the "fledge" to "protected-audience" in the headers when we wanted to cleanup the reference to "fledge" in spec after the team is renamed. You can see all headers (including the other two) were changed to "X-protected-audience" from "X-Fledge" in the spec, which was wrong anyways.

Reply:
Acknowledged, thanks for pointing out that we don't accept the new header name.